### PR TITLE
Pass the `--gimbal` option through to legacy SITL

### PR DESCRIPTION
**Pull request by wsilva32**
* requires that apm is built with the make sitl-mount target

### CODE REVIEW

**Comment by mrpollo** [comment]
I take it none of the dronekit hosted binaries support this right?

File: dronekit_sitl/__init__.py [hunk 1]
```diff
@@ -207,6 +207,7 @@ def launch(self, args, verbose=False, await_ready=False, restart=False, wd=None,
             parser.add_argument('--rate')
             parser.add_argument('--model')
             parser.add_argument('-C', action='store_true')
+            parser.add_argument('--gimbal', action='store_true')
```
Comment (mrpollo) [inline dronekit_sitl/__init__.py]: would you mind adding the pre-requisites for this to work on `help` please?

**Comment by wsilva32** [comment]
the --gimbal will enable gimbal.py in sim_wrapper (legacy SITL). It is not reverse compatible with older APM versions.

**Comment by mrpollo** [comment]
Thanks :) 👍

**Comment by peterbarker** [comment]
Should probably cherry-pick this in:

https://github.com/dronekit/dronekit-sitl/commit/a7d76eca557e60c408d223d31539260d47a5d171

**Comment by wsilva32** [comment]
Seems like a good idea @peterbarker. Were you planning on merging that into master at some point in a separate PR?

**Pull request merged**